hack/preview.sh

```diff
@@ -49,6 +49,18 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+oc patch ArgoCD/openshift-gitops -n openshift-gitops --type=merge --patch-file=/dev/stdin << EOF
+spec:
+  repo:
+    env:
+      - name: ARGOCD_EXEC_TIMEOUT
+        value: 5m
```

Comment:
Should we move this in hack/deploy-argocd.sh script? That other script is deploying argoCD and configuring.

Reply:
good point

Reply:
@hugares  cool, I've put it to `preview.sh`, because I was not sure whether `hack/deploy-argocd.sh` is used only in dev environment. happy to move it to `hack/deploy-argocd.sh`

Reply:
yes it is for dev, it is used by bootstap-cluster.sh only AFAIK. I know for sure it is not used in staging or prod, we deploy argocd using app-interface for those environments